Sources/App/Models/Session/Session.swift

```diff
@@ -1,8 +1,10 @@
 import Vapor
 import FluentProvider
+import Foundation
+import Crypto
 
 // sourcery: AutoModelGeneratable
-// sourcery: fromJSON, toJSON, Preparation
+// sourcery: Preparation, Timestampable
```

Comment:
I think you could return toJSON and use `// sourcery: ignoreInJSON`  to `userId` look at `Place` e.g.

Reply:
@ditansu Yes, he could do this, but I see that there's no "id" also in custom implementation. because we add "id" field in JSON by default in template...

Reply:
I'll leave it as is to use it without `id`.

Reply:
Ok